### PR TITLE
Task#7479 - styling sidebar

### PR DESCRIPTION
**Pull request by HeorhiiBoliuk**
https://redmine.devbranch.work/issues/7479/time_entries/new

### CODE REVIEW

File: .ddev/.ddev-docker-compose-full.yaml
Comment (KuzyaWkk) [inline .ddev/.ddev-docker-compose-full.yaml:1]: .ddev/.ddev-docker-compose-full in git?

File: web/themes/custom/my_awesome_theme/components/00-base/00-defaults/_03-mixins.scss [hunk 1]
```diff
@@ -47,6 +47,20 @@
...
+@mixin title_block{
+  text-align: left;
+  font-family: 'Roboto',sans-serif;
+  font-size: 14px;
+  line-height: 1.4;
+  margin-top: 10px;
+  margin-bottom: 26px;
+  border-bottom: 2px solid var(--td_header_color,var(--td_black,#222222));
+  span{
+    color: #ffffff;
```
Comment (KuzyaWkk) [inline web/themes/custom/my_awesome_theme/components/00-base/00-defaults/_03-mixins.scss:59]: Why are you don`t use variables for color?
E.g. $color-main: #fff;
color: $color-main;

File: web/themes/custom/my_awesome_theme/templates/block/block--views-block--most-popular-block-block-1.html.twig
Comment (KuzyaWkk) [inline web/themes/custom/my_awesome_theme/templates/block/block--views-block--most-popular-block-block-1.html.twig:1]: Many changes in twig templates have nothing to do with styling. It is better to indicate in the comments that such changes have been added, and why they have been added

**Review by KuzyaWkk — CHANGES_REQUESTED** [review]
Add a comment on why you made changes to the twig template. And check if .ddev-docker-compose-full.yaml shouldn't be in .gitignore

File: web/themes/custom/my_awesome_theme/templates/block/block--views-block--most-popular-block-block-1.html.twig
Comment (HeorhiiBoliuk) [reply web/themes/custom/my_awesome_theme/templates/block/block--views-block--most-popular-block-block-1.html.twig:1]: Next time I will comment on the changes that not involved in a task overview.

Comment (HeorhiiBoliuk) [reply web/themes/custom/my_awesome_theme/templates/block/block--views-block--most-popular-block-block-1.html.twig:1]: In the last branch I forgot to add a node URL. Add some changes in this branch

**Review by KuzyaWkk — APPROVED** [review]

**Comment by KuzyaWkk** [comment]
@lavrenchuk-dev

**Review by lavrenchuk-dev — APPROVED** [review]

**Pull request merged**